controller/cmd/service-mirror/cluster_watcher.go

```diff
@@ -458,6 +501,17 @@ func remapRemoteServicePorts(ports []corev1.ServicePort) []corev1.ServicePort {
 	return newPorts
 }
 
+func endpointAddressesToIps(addrs []corev1.EndpointAddress) []string {
```

Comment:
every time I read go code like this I cry sorrowful tears of yearning for `.map(_.IP)`

Reply:
every time I have to write code like this I feel the golang creators are actually praniking me (and you because you have to read it... )